features/interactivity.yml

```
@@ -0,0 +1,3 @@
+name: interactivity
+description: The CSS `interactivity` property specifies whether an element and its descendants are inert or not.
```

```suggestion
description: The CSS `interactivity` property makes an element and its descendants inert, like when using the `inert` HTML attribute. Inert elements can't be focused or clicked, their text can't be selected or found using the browser's find-in-page feature.
```

I thought it'd be good to give more details about what inert means, and to mention the related html attribute as well.

---

I added the more detailed suggestion @jamesnw suggested.